### DISCUSSION TITLE
Getting a lot of stats_timestamp ... is later than collector's time ... for database 0

### DISCUSSION THREAD

**Discussion by refrain** [teslamate-org/teslamate#2317] [Q&A]
Running teslamate version 1.25.0 in docker, getting a lot of these in the logs:

atabase_1   | 1970-01-01 00:00:00.000 UTC [30] LOG:  stats_timestamp 1971-05-10 06:32:39.045168+00 is later than collector's time 1970-06-24 03:12:48+00 for database 0
database_1   | 1970-01-01 00:00:00.000 UTC [29] LOG:  stats collector's time 1971-05-10 06:33:59.045168+00 is later than backend local time 1970-01-01 00:00:01+00
database_1   | 1970-01-01 00:00:00.000 UTC [30] LOG:  stats_timestamp 1971-05-10 06:33:59.045168+00 is later than collector's time 1970-06-24 03:12:48+00 for database 0
database_1   | 1970-01-01 00:00:00.000 UTC [29] LOG:  stats collector's time 1971-05-10 06:32:39.045168+00 is later than backend local time 1970-01-01 00:00:01+00
database_1   | 1970-01-01 00:00:00.000 UTC [30] LOG:  stats_timestamp 1971-05-10 06:32:39.045168+00 is later than collector's time 1970-06-24 03:12:48+00 for database 0

Any idea how to fix this?

**Comment by hbaerten** [comment]
Same here:

```
{"log":"1970-01-01 00:00:00.000 UTC [29] LOG:  stats collector's time 2005-11-28 02:25:14.105368+00 is later than backend local time 1970-01-01 00:00:01+00\n","stream":"stderr","time":"2022-03-02T08:48:48.222127636Z"}
{"log":"1970-01-01 00:00:00.000 UTC [30] LOG:  stats_timestamp 2005-11-28 02:25:14.105368+00 is later than collector's time 1970-01-01 00:35:26.662272+00 for database 0\n","stream":"stderr","time":"2022-03-02T08:48:48.222345656Z"}

```
The much bigger problem I have is that this is logged 10 times per second and so in 2 months time the postgres log grew to 16GB and filled up my root filesystem.
So apart from fixing this error, is there also a recommended way to prevent the (docker container) logfile from growing indefinitely?
